packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.tsx

```diff
@@ -89,19 +87,18 @@ export const NavigationPropertyTargetSelector = forwardRef<NavigationPropertyTar
         tabSelectsValue={false}
         loadingMessage={() => translate("navigation-property-editor.loading-target-instances")}
         styles={{
-          control: () => ({ height: "27px" }),
-          container: () => ({ width: "auto" }),
-          valueContainer: () => ({ height: "27px", ["--_iui-select-padding-block"]: 0, ["--_iui-select-min-height"]: "var(--iui-component-height-small)" }),
-          menu: () => ({ position: "absolute", zIndex: 9999, width }),
-          menuList: (style: any) => ({ ...style, padding: 0 }),
-          option: () => ({ whiteSpace: "nowrap", width: "max-content", minWidth: "100%" }),
-          dropdownIndicator: () => ({ backgroundColor: "var(--iui-color-background)" }),
+          control: () => ({}),
+          container: () => ({}),
```

Comment:
do they need to be specified if they're returning empty styles?

Reply:
Yes. These are for removing default styling.